views/beers.hbs

```diff
@@ -0,0 +1,8 @@
+
+<div>
+  {{#each beersFromApi}}
+{{>beerpartial}}
+  <a href="h"></a>
```

Comment:
This href is probably here by mistake. After the work is done, give a look in your code and check for this type of occurrences. It's not breaking anything in your code, but it's a bad pratice.